Source/ARTRealtime.m

```diff
@@ -1637,7 +1637,7 @@ - (void)realtimeTransportFailed:(id<ARTRealtimeTransport>)transport withError:(A
     
     if ([self shouldRetryWithFallback:transportError]) {
         ARTLogDebug(self.logger, @"R:%p host is down; can retry with fallback host", self);
-        if (!_fallbacks && [transportError.url.host isEqualToString:[ARTDefault realtimeHost]]) {
+        if (!_fallbacks && [transportError.url.host hasSuffix:[ARTDefault realtimeHost]]) { // this checks if host is a default one regardless environment used, RTN17b
```

Comment:
Do you not think that instead of this check, it might be better for us to actually implement the check that's described in RTN17b? For example, this check doesn't take into account whether the user has changed the port, or whether they've provided an array of `fallbackHosts`, right? (Meaning that we probably aren't satisfying the sub-points of RTN17b, and we should fix and test that too.)

Reply:
done https://github.com/ably/ably-cocoa/pull/2017/commits/2a9b845dad248acaa9cfc2584bd3a52481749f63